benchmarks/benchmark/tools/latency-profile/manifest-templates/latency-profile-generator.yaml.tpl

```diff
@@ -0,0 +1,43 @@
+apiVersion: "apps/v1"
+kind: "Deployment"
+metadata:
+  name: lantency-profile-generator
+  namespace: ${namespace}
+  labels:
+    name: lantency-profile-generator
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: lantency-profile-generator
+  template:
+    metadata:
+      labels:
+        app: lantency-profile-generator
+        examples.ai.gke.io/source: ai-on-gke-benchmarks
+    spec:
+      serviceAccountName: ${latency_profile_kubernetes_service_account}
+      containers:
+        - name: lantency-profile-generator
+          image: ${artifact_registry}/latency-profile:latest
+          command: ["bash", "-c", "./latency_throughput_curve.sh"]
+          env:
+            - name: TOKENIZER
+              value: ${tokenizer}
+            - name: IP
+              value: ${inference_server_service}
+            - name: BACKEND
+              value: ${inference_server_framework}
+            - name: INPUT_LENGTH
+              value: ${max_prompt_len}
+            - name: OUTPUT_LENGTH
+              value: ${max_output_len}
+            - name: OUTPUT_BUCKET
+              value: ${output_bucket}
+%{ for hugging_face_token_secret in hugging_face_token_secret_list ~}
```

Comment:
Since you have support for k8s_hf_secret, you'll need to add something like this code here to support it in this spec:

https://github.com/GoogleCloudPlatform/ai-on-gke/blob/main/benchmarks/benchmark/tools/locust-load-inference/manifest-templates/locust-worker-controller.yaml.tpl#L57:L63

Reply:
looks like this was addressed, resolving :)